cardano-crypto-class/cardano-crypto-class.cabal

```
@@ -130,7 +130,7 @@ library
     build-depends:
       integer-gmp
   pkgconfig-depends:
-    libblst,
+    libblst >=0.1.14,
```

:wink: 
```suggestion
    libblst >=0.3.14,
```
You also had the same mistake in a few places in the PR description, which I've taken the liberty of fixing